### PR TITLE
docs: add release notes for 0.19.2

### PR DESCRIPTION
**Pull request by mackrorysd**
Add release notes for 0.19.2.

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -10,6 +10,59 @@
...
+
+-  CLI: ``det trial describe --json`` output format has changed. Fixed a bug where ``det trial
+   describe --json --metrics`` would fail for trials with a very large number of steps.
+
+-  CLI: ``det job list`` will now return all jobs by default instead of a single API results page.
+   Use ``--pages=1`` option for the old behavior.
+
+**Bug Fixes**
+
+-  Kubernetes: Fixed an issue where restoring a job in into a Kubernetes set up could crash the
```
Comment (sirredbeard) [inline docs/release-notes.rst:37]: nit: `in into` is repetitive

File: docs/release-notes.rst [hunk 2]
```diff
@@ -10,6 +10,59 @@
...
+   killed.
+
+**Improvements**
+
+-  API: `GetTrialWorkloads` can now optionally include per-batch metrics when
+   ``includeBatchMetrics`` query parameter is set.
+
+**New Features**
+
+-  Cluster: Determined Enterprise Edition can now be deployed on a Slurm cluster. When using Slurm,
```
Comment (vishnu2kmohan) [inline docs/release-notes.rst:60]:
```suggestion
-  Cluster: The enterprise edition of Determined ([HPE Machine Learning Development Environment](https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html)), can now be deployed on a Slurm cluster. When using Slurm,
```

File: docs/release-notes.rst [hunk 3]
```diff
@@ -10,6 +10,59 @@
...
+   Determined delegates all job scheduling and prioritization to the Slurm workload manager. This
+   integration enables existing Slurm workloads and Determined workloads to coexist and Determined
```
Comment (vishnu2kmohan) [inline docs/release-notes.rst:62]:
```suggestion
   integration enables existing Slurm workloads and Determined workloads to coexist and ```

File: docs/release-notes.rst [hunk 4]
```diff
@@ -10,6 +10,59 @@
...
+   workloads to access all of the advanced capabilities of the Slurm workload manager. The
```
Comment (vishnu2kmohan) [inline docs/release-notes.rst:63]:
```suggestion
   access all of the advanced capabilities of the Slurm workload manager. The
```

File: docs/release-notes.rst [hunk 5]
```diff
@@ -10,6 +10,59 @@
...
+   Determined Slurm integration can use either Singularity or PodMan for the container runtime.
```
Comment (vishnu2kmohan) [inline docs/release-notes.rst:64]:
```suggestion
   Determined Slurm integration can use either Singularity or Podman for the container runtime.
```

**Review by vishnu2kmohan — CHANGES_REQUESTED** [review]
Minor tweaks to the release notes announcing the Slurm integration for EE.

**Review by vishnu2kmohan — APPROVED** [review]
Thanks, @mackrorysd!

**Comment by mackrorysd** [comment]
Merging despite the CI failures: test-e2e is failing because of a Docker Hub outage only (and the docs don't affect these tests). Linting is failing because of an API change already in master (so the compatibility check doesn't apply) and because of a genuine Python Linting issue that was fixed after branching which I won't cherry-pick at this point.

**Pull request merged**